Deploy/ActivateSSL.ps1

```diff
@@ -41,21 +41,20 @@ param(
 # Helper var
 $success = $True
 $terraformFolder = "SSLActivation"
-$iaCFolder = "IaC"
 $webAppsVariableFile = "$(Get-ScriptPath)/$terraformFolder/webAppVariable.tfvars.json"
 
 # Tell who you are (See HelperFunction.ps1)
 Write-WhoIAm
 
 # 1. Read values from Terraform IaC run (Bot deployment scripts)
 Write-Host "## 1. Read values from Terraform IaC run (Bot deployment scripts)"
-$content = '{ "azure_webApps" : ' + $(terraform output -state="$(Get-ScriptPath)/$iaCFolder/terraform.tfstate" -json webAppAccounts) + '}'
+$content = '{ "azure_webApps" : ' + $(Get-TerraformOutput("webAppAccounts")) + '}'
```

Comment:
It is just question! Now do we store tfstates under webapp which is the place bot deployed as well? Isn't there any security concern cause webapp is publicly opened?

Reply:
No, we store the Terraform State in a separate storage account created in the new InitTerraform.ps1 script https://github.com/h2floh/GeoDistributedAzureBot/blob/420458620b71837701d7b6bf5813a62977dfa262/Deploy/InitTerraform.ps1#L66

 The above code will retrieve the Terraform Output details for the WebApps, from the initial run from DeployInfrastructure.ps1, the info is stored as Terraform State file in the Storage Account.